perf-common/src/main/java/com/github/projectflink/common/ThroughputMean.java

```diff
@@ -0,0 +1,27 @@
+package com.github.projectflink.common;
+
+public class ThroughputMean {
+	double max = Double.MIN_VALUE;
+	double elementsSum;
+	double timeSum;
+	long count;
+
+	public void addMeassurement(double elements, double time) {
+		max = Math.max(max, elements / time);
+		elementsSum += elements;
+		timeSum += time;
```

Comment:
Isn't this sum a bit risky in case of very long runs with a lot of elements being reported? It may come to the point where the actual sum is that big so that a single `elements` is not big enough to add anything to the `double` value but lots of those would have added some. I guess, this is what `org.apache.commons.math3.stat.descriptive.moment.Mean#evaluate(double[], double[], int, int)` is trying to improve with the correction factor (a bit hard to get their intention without access to the mentioned paper).